app/models/package.js

```diff
@@ -48,6 +48,8 @@ export default DS.Model.extend({
     return this.get('packageType.name');
   }),
 
+  changedNotes: Ember.computed.alias("notes"),
```

Comment:
@abulsayyad123 , it doesn't make any sense to me to alias a model attribute in model itself. 
What's the issue with using `model.notes`?

Reply:
It is giving error related to binding which says that you cannot edit binded variable i.e notes. That is why i had alias notes variable.